Sources/CompressNIO/zlib/Zlib.swift

```diff
@@ -138,7 +138,7 @@ public struct ZlibConfiguration: Sendable {
 
 /// Compressor using Zlib
 public final class ZlibCompressor {
-    var stream: z_stream
+    var stream: UnsafeMutablePointer<z_stream>
```

Comment:
Can we add a comment detailing why we need to use an `UnsafeMutablePointer`

Reply:
Ok, added comment to both places.

Reply:
Thanks. This is great. It might be worthwhile pointing this out in the swift-nio-extras repository. They have some request/response compression channel handlers use Zlib in a similar manner.

Reply:
I was already on the fence but then Apple decided that they want to play both sides (donating to Trump's inauguration, returning to advertising on X versus https://www.swift.org/diversity/ ), so I decided to not do any free QA for them unless absolutely necessary.

Reply:
Fair enough